src/objdictgen/node.py

```diff
@@ -984,7 +1010,7 @@ def GetTypeList(self):
         list_ = Find.TypeList(MAPPING_DICTIONARY)
         for mapping in self.GetMappings():
             list_.extend(Find.TypeList(mapping))
-        return ",".join(sorted(list_))
+        return list_
```

Comment:
Why can we use true lists here instead of comma separated string? Is it because of new wx UI handling?

Reply:
Correct, the latest implementation of GridCellChoiceEditor expects a true list -> https://docs.wxpython.org/wx.grid.GridCellChoiceEditor.html?highlight=gridcellchoiceeditor#wx.grid.GridCellChoiceEditor

From my investigation into the previous implementation this method is exclusively used when generating the UI drop downs and therefore I thought it appropriate to modify the base functions in node.py.